src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_AddImport.cs

```diff
@@ -58,31 +58,25 @@ public SymbolSearchService(CodeAnalysisService codeAnalysisService)
                 this.codeAnalysisService = codeAnalysisService;
             }
 
-            public async Task<IList<PackageWithTypeResult>> FindPackagesWithTypeAsync(
+            public Task<IList<PackageWithTypeResult>> FindPackagesWithTypeAsync(
```

Comment:
:memo: This one makes me uncomfortable (loses the name `FindPackagesWithTypeAsync` in the stack trace, plus this is an OOP call so the overhead of the async state machine is minimal in comparison.